src/Bot/commands/commandsReplies.ts

```diff
@@ -16,11 +16,11 @@ const graphQLClient = new GraphQLClient(config.graphqlAPI);
 
 export const lookupReply = async (interaction: CommandInteraction) => {
   try {
-    const usernameArg = interaction.options.get("username");
+    const usernameOption = interaction.options.get("username");
```

Comment:
https://discordjs.guide/slash-commands/parsing-options.html#command-options use `getString` instead to get the string. Then, either add a default value here i.e. `.getString(..) ?? "..."` or if you are sure it will not be undefined, then `.getString(..) as string` so we don't have to check that this exists everywhere when using, just check it once.

Reply:
[`.getString` doesn't exist on type `CommandInteraction['options']`](https://github.com/discordjs/discord.js/blob/5bfcf93e29ab734da670d9955f38163268d6a675/packages/discord.js/typings/index.d.ts#L520-L534).